app/models.py

```diff
@@ -1390,6 +1390,10 @@ class NotificationAllTimeView(db.Model):
 
     __tablename__ = "notifications_all_time_view"
 
+    # Tell alembic not to create this as a table. We have a migration where we manually set this up as a view.
+    # This is custom logic we apply - not built-in logic. See `migrations/env.py`
+    __table_args__ = {"info": {"managed": False}}
```

Comment:
i don't love `managed` here, i dont think it's immediately apparent that that refers to alembic. how about `managed_by_alembic`?